dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/tcp/handler/DssTcpChannelInitializer.java

```diff
@@ -0,0 +1,29 @@
+package io.github.ztkmkoo.dss.core.network.tcp.handler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.DelimiterBasedFrameDecoder;
+import io.netty.handler.codec.Delimiters;
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+
+public class DssTcpChannelInitializer extends ChannelInitializer<SocketChannel> {
+
+    private final Logger logger = LoggerFactory.getLogger(DssTcpChannelInitializer.class);
+
+    @Override
+    protected void initChannel(SocketChannel ch) throws Exception {
+        logger.info("Try to initChannel");
+
+        final ChannelPipeline p = ch.pipeline();
+
+        p.addLast(new DelimiterBasedFrameDecoder(8192, Delimiters.lineDelimiter()));
+        p.addLast(new StringDecoder());
+        p.addLast(new StringEncoder());
```

Comment:
If using String, you'd better config encoding

Reply:
I'm not sure if we need other encoding? (I think most case use UTF-8)

Reply:
At first, there isn't special tends using String Encoder.
I found StringEncoder and ByteArrayEncoder. which one would you recommend to use? 

if i use string encoder i can add encoding config like this. 

```suggestion
        p.addLast(new StringDecoder(CharsetUtil.UTF_8));
        p.addLast(new StringEncoder(CharsetUtil.UTF_8));
```

Reply:
Ok, 
Let's set encoding as UTF-8.

And then, let's talk bout StringDecoder and SimpleChannelInboundHandler<String>.
Actually we need to deserialize the client's request(byte array) to some object and handle the object in service actor.
And using StringDecoder / StringEncoder means deserializing to string first and convert this deserialized string to object. It's the common choice and human readable.

But i'm afraid it would not work in some cases. Give you feedback after some tests.